commands/releases/index.js

```diff
@@ -11,11 +11,10 @@ function * run (context, heroku) {
   let descriptionWithStatus = function (d, r) {
     const width = () => process.stdout.columns > 80 ? process.stdout.columns : 80
     const trunc = (s, l) => truncate(s, {length: width() - (60 + l), omission: '…'})
-
-    let status = statusHelper(r.status)
+    let status = statusHelper.description(r, runningRelease, runningSlug)
     let sc = ''
-    if (status.content !== undefined) {
-      sc = cli.color[status.color](status.content)
+    if (status !== undefined) {
```

Comment:
is it necessary to check if it's exactly undefined? or would `if (status)` be enough?